css/types/frequency.json

```diff
@@ -7,10 +7,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/frequency",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": false
```

Comment:
What's the evidence that this doesn't work in Chrome. I can find all of it in our source files.

Reply:
What does it do then if anything?
As it says in the docs: " no CSS property is using it at the moment."

Reply:
It seems to be related to Typed OM. https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/css/css_properties.json5?q=frequency+file:%5Esrc/third_party/blink/renderer/core/css/+package:%5Echromium$&dr=C&l=252

Reply:
Hm, I can't find an API using it. `CSSPrimitiveValue` mentions it but according to our data it is unsupported as well: https://developer.mozilla.org/en-US/docs/Web/API/CSSPrimitiveValue#Browser_compatibility

Reply:
So, in Chrome, you can run `CSS.kHz("2.0")` and `new CSSUnitValue(2.0, "hz")`. I'm not sure if the CSS frequency type refers to these things or we should rather have that compat info within these APIs? What do you think?

Reply:
Frankly, I don't know enough about this area. Let me see if I can enlist some help.

Reply:
Yeah, it’s an esoteric difference and I’m not sure what is better to list here. Chrome’s parser _does_ support frequency and the unit suffixes (as shown by the WPT), but CSS doesn’t have any props that use frequency 🤷‍♂

Reply:
I think we should go with `false` like this PR proposes. 
First, for consistency with the other browsers' support data (maybe other browsers parse this, too, but we don't indicate that right now either).  
And second, to not mislead web developers about a frequency unit that they can't actually use. 
Does that sound reasonable?